app/models/grda_warehouse/warehouse_reports/outflow_report.rb

```diff
@@ -85,35 +85,46 @@ def outflow_to_housing
       chart_end_date = @filter.end.end_of_month
       months = (chart_start_date.to_date..chart_end_date.to_date).map { |m| m.strftime('%b %Y') }.uniq
 
-      clients_to_ph = exits_scope(start_date: chart_start_date, end_date: chart_end_date).
+      hoh_to_ph = exits_scope(start_date: chart_start_date, end_date: chart_end_date).
+        heads_of_households.
         where(destination: HUD.permanent_destinations).
         distinct.
         select(:client_id, :last_date_in_program).
         index_by(&:client_id).values.
         group_by{ |x| x.last_date_in_program.end_of_month }
 
-      clients_to_ph_count = {}
-      months.each { |month| clients_to_ph_count[month] = 0 }
-      clients_to_ph_count.merge!(clients_to_ph.map { |k, v| [k.strftime('%b %Y'), v.size] }.to_h)
+      hoh_to_ph_count = {}
+      months.each { |month| hoh_to_ph_count[month] = 0 }
+      hoh_to_ph_count.merge!(hoh_to_ph.map { |k, v| [k.strftime('%b %Y'), v.size] }.to_h)
 
-      clients_to_stabilization = housed_scope.
+      hoh_to_stabilization = housed_scope.
+        heads_of_households.
         where(project_type: [3, 9, 10, 13]). # PSH or RRH
         entering_stabilization(start_date: chart_start_date, end_date: chart_end_date).
         distinct.
         select(:client_id, :housed_date).
         index_by(&:client_id).values.
         group_by{ |x| x.housed_date.end_of_month }
 
-      clients_to_stabilization_count = {}
-      months.each { |month| clients_to_stabilization_count[month] = 0 }
-      clients_to_stabilization_count.merge!(clients_to_stabilization.map { |k, v| [k.strftime('%b %Y'), v.size] }.to_h)
+      hoh_to_stabilization_count = {}
+      months.each { |month| hoh_to_stabilization_count[month] = 0 }
+      hoh_to_stabilization_count.merge!(hoh_to_stabilization.map { |k, v| [k.strftime('%b %Y'), v.size] }.to_h)
+
+      hoh_exits_to_ph = (hoh_to_ph.keys | hoh_to_stabilization.keys).map do |month|
+        [month, (Array.wrap(hoh_to_ph[month]) + Array.wrap(hoh_to_stabilization[month])).uniq(&:client_id)]
```

Comment:
I think we'll need to be a bit careful about nils here.  If there are no hoh_to_ph[month] in a given month you'll end up with `[nil] + [...]` and I think calling `uniq(&:client_id)` will probably fail.  maybe:
```
[
  month, 
  (Array.wrap(hoh_to_ph[month]).compact + Array.wrap(hoh_to_stabilization[month].compact)).uniq(&:client_id),
]
```

Reply:
In my test data there were some empty months for hoh_to_stabilization[month]. Using `Array.wrap` seemed to address the problem, but it looks like using `compact` will be safer.